src/components/SearchBar.vue

```diff
@@ -123,13 +163,24 @@ export default {
       }
       return this.$t('gutenberg_search_placeholder');
     },
-    handleEnter() {
-      if (!this.searchQuery || this.searchQuery.trim() === '') {
-        const fallbackKeyword =
-          this.randomKeywords?.[0] || this.$t('gutenberg_search_placeholder');
-        this.$emit('input', fallbackKeyword);
+    handleInput(event) {
+      this.currentSearchQuery = event.target.value;
```

Comment:
I think we don't need to set `currentSearchQuery` explicitly in `@input` if it is in `v-model`